Clarify Android platform-view mechanics and HCPP requirements (#13705)
flutter/website: sites/docs/src/content/platform-integration/android/platform-views.md

## Hybrid composition {: #hybrid-composition }

Platform views are rendered as they normally are,
directly in the Android view hierarchy.

> Flutter content is rendered into `ImageReader`s and painted by
> `FlutterImageView`s that sit in the same window as the platform views,

Reviewer: We try to avoid making code font spans possessive. Would something like the following work/be accurate? It also switches to active voice.

```suggestion
Flutter renders its content into `ImageReader` buffers then
`FlutterImageView` instances paint those buffers
in the same window as the platform views,
```

* **Android API 34 or later**: Required for native transaction
  synchronization capabilities.
* **Impeller with the Vulkan backend**: The device must be capable of

>   rendering with Vulkan. Impeller also has an OpenGLES backend, and

Reviewer: ```suggestion
  rendering with Vulkan. Impeller also has an OpenGL ES backend, and
```

  falls back to it on devices without usable Vulkan support.

>   HCPP is not available in that configuration.

Reviewer: ```suggestion
  HCPP isn't available in that configuration.
```

Platform views in Flutter come with performance trade-offs.

In a typical Flutter app,
Flutter rasterizes frames on a dedicated raster thread,
while platform code, such as plugins and Android views,
runs on the platform thread, which is the Android main thread.
This separation keeps Flutter rendering fast and fluid,

> as the platform thread is not blocked by rasterization work.

Reviewer: ```suggestion
as the platform thread isn't blocked by rasterization work.
```

> However, when a platform view is rendered on Android using
> hybrid composition,

Reviewer: Adjust the break here:

```suggestion
However, when a platform view is
rendered on Android using hybrid composition,
```
